docs/en/advanced_tutorials/visualization.md

```diff
@@ -1,3 +1,390 @@
+import mmcv
+from mmengine.visualization import Visualizer
+
+image = mmcv.imread('docs/en/_static/image/cat_dog.png', channel_order='rgb')
+visualizer = Visualizer(image=image)
+# single bbox formatted as [xyxy]
+visualizer.draw_bboxes(torch.tensor([72, 13, 179, 147]))
+# draw multiple bboxes
+visualizer.draw_bboxes(torch.tensor([[33, 120, 209, 220], [72, 13, 179, 147]]))
+visualizer.show()
+```
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/17425982/186052649-8611ae43-1bb9-46e8-b6a1-dfc5063407c7.png" width="400"/>
+</div>
+
+```python
+visualizer.set_image(image=image)
+visualizer.draw_texts("cat and dog", torch.tensor([10, 20]))
+visualizer.show()
+```
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/17425982/186052726-bd8f1571-b34b-471a-9876-9f0ae8c4e2be.png" width="400"/>
+</div>
+
+You can also customize things like color and width using the parameters in each API.
+
+```python
+visualizer.set_image(image=image)
+visualizer.draw_bboxes(torch.tensor([72, 13, 179, 147]), edge_colors='r', line_widths=3)
+visualizer.draw_bboxes(torch.tensor([[33, 120, 209, 220]]),line_styles='--')
+visualizer.show()
+```
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/17425982/186053154-ddd9d6ec-56e0-45cc-86a8-b349812ba2e7.png" width="400"/>
+</div>
+
+2. Overlay display
+
+These APIs can be called multiple times to get an overlay result.
+
+```python
+visualizer.set_image(image=image)
+visualizer.draw_bboxes(torch.tensor([[33, 120, 209, 220], [72, 13, 179, 147]]))
+visualizer.draw_texts("cat and dog",
+                      torch.tensor([10, 20])).draw_circles(torch.tensor([40, 50]), torch.tensor([20]))
+visualizer.show()
+```
+
+<div align=center>
+<img src="https://user-images.githubusercontent.com/17425982/186053209-de5f57e5-4ccb-45af-9370-1788d257123b.png" width="400"/>
+</div>
+
+## Feature Map Visualization
+
+Feature map visualization has many functions. Currently, we only support single feature map visualization.
+
+```python
+@staticmethod
+def draw_featmap(featmap: torch.Tensor, # input format must be CHW
+                 overlaid_image: Optional[np.ndarray] = None, # if image data is input at the same time, the feature map will be overlaid on the image
+                 channel_reduction: Optional[str] = 'squeeze_mean', # strategy to compress multiple channels into a single channel
+                 topk: int = 10, # topk feature maps to show
+                 arrangement: Tuple[int, int] = (5, 2), # the layout when multiple channels are expanded into multiple images
+                 resize_shape：Optional[tuple] = None, # scale the feature map
+                 alpha: float = 0.5) -> np.ndarray: # overlay ratio between input image and generated feature map
+```
+
+The main features can be concluded as follows:
+
+- As the input Tensor usually includes multiple channels, `channel_reduction` can compress them into a single channel and overlay the result to the image.
```

Comment:
I think it's better to use the terminology `reduce`/`reduction` here instead of `compress`/`compression`

Reply:
> I think it's better to use the terminology `reduce`/`reduction` here instead of `compress`/`compression`

agreed.